FWCore/MessageService/src/ELoutput.cc

```diff
@@ -95,7 +95,7 @@ namespace edm {
     // Useful function:
     // ----------------------------------------------------------------------
 
-    static ELstring formatTime(const time_t t) {  // Change log 7
+    static std::string formatTime(const time_t t) {  // Change log 7
 
       static char const dummy[] = "dd-Mon-yyyy hh:mm:ss TZN     ";  // Change log 7 for length only
```

Comment:
How about `constexpr` instead of `static`?

Reply:
I agree